docker/dockerfiles/Dockerfile_es

```diff
@@ -1,4 +0,0 @@
-FROM elasticsearch:1.7.4
-
-# Install HEAD plugin
-RUN plugin -i mobz/elasticsearch-head
```

Comment:
Yes, sorry about that. Closing the loop for others who may find this: #12083 and #12128